app/graphql/types/objects/consultation_partner_response/base.rb

```diff
@@ -0,0 +1,17 @@
+module Types
+	module Objects
+		module ConsultationPartnerResponse
+			class Base < BaseObject
+				graphql_name "ConsultationPartnerResponseType"
+				field :organisation_id, 												Integer, nil, null: false
```

Comment:
@austinjb32 if we have added organisation as field then we don't need organisation_id, FE can get id from organisation object

Reply:
@Mahaveer1141 Yes that so correct. But can you look the way i have taken the value. I have taken it from organisation_id itself. Can you suggest a work around for that?

Reply:
there's no problem there, you can just remove this field.
you can also pass organisation model instance directly.